Dockerfile

```diff
@@ -46,6 +44,9 @@ RUN export BUILD_NUMBER=${BUILD_NUMBER} && \
 
 RUN npm prune --no-audit --production
 
+RUN apt-get autoremove -y && \
```

Comment:
no point doing it here as the base image will already have the dependencies in it so contribute to the final image size - you have to do it as part of the update/upgrade stage...

Reply:
![Screenshot 2023-03-24 at 10 56 07](https://user-images.githubusercontent.com/92578842/227511595-3ffdfb1a-a7fe-4b89-8552-183082d9377c.png)
this is a current size of the image comparing to the last successful one